meetings/2024-04-10.md

```diff
@@ -0,0 +1,77 @@
+# Node.js Technical Steering Committee (TSC) Meeting 2024-04-10
+
+## Links
+
+* **Recording**: https://www.youtube.com/watch?v=_tV3RODb8L8
+* **GitHub Issue**: https://github.com/nodejs/TSC/issues/1526
+* **Minutes Google Doc**: https://docs.google.com/document/d/1k78cpfIZRpZU9FGRCeGMFzPms9IwnfVrcJvSR9NDmb4/edit
+
+## Present
+
+* Antoine du Hamel @aduh95 (voting member)
+* Chengzhong Wu @legendecas (voting member)
+* Geoffrey Booth @GeoffreyBooth (voting member)
+* Gireesh Punathil @gireeshpunathil (voting member)
+* James Snell @jasnell (voting member)
+* Joyee Cheung @joyeecheung (voting member)
+* Michaël Zasso @targos (voting member)
+* Moshe Atlow @MoLow (voting member)
+* Paolo Insogna @ShogunPanda (voting member)
+* Rafael Gonzaga @RafaelGSS (voting member)
+* Rich Trott @Trott (regular member)
+* Richard Lau @richardlau (voting member)
+* Robert Nagy @ronag (voting member)
+* Ruy Adorno @ruyadorno (voting member)
+* Yagiz Nizipli @anonrig (voting member)
+
+## Agenda
+
+### Announcements
+
+* Package maintenance team met and drafted some goals related to version management: https://github.com/nodejs/package-maintenance/pull/597/files?short_path=6db9788#diff-6db9788007092f5808d709367ce74c04e7b45b842170823c93f348ab71efd827
+* Raz improved AbortSignal.abort() creation which impacts test runner as well - https://github.com/nodejs/node/pull/52408
+* Daniel Lemire’s work on simdutf resulted in great improvements on base64 operations such as https://github.com/nodejs/node/pull/52428.
+* Security release has been delayed and should be out in a few hours.
+
+### CPC and Board Meeting Updates
+
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/node
+
+* deps,lib,src: add experimental web storage [#52435](https://github.com/nodejs/node/pull/52435)
+  * We discussed and came to conclusion to proceed with the pull-request. We have no objections for it to land, other than existing collaborator comments.
+* doc: add policy for “placeholder” executables [#52107](https://github.com/nodejs/node/pull/52107)
+  * A vote has been started - https://github.com/nodejs/TSC/pull/1527
```

Comment:
Is this link correct? It says "Open Corepack Vote", not "placholder vote"

Reply:
I believe that is the right link